clients/cli/src/main/java/org/apache/gravitino/cli/FullName.java

```diff
@@ -29,6 +29,8 @@ public class FullName {
   private final CommandLine line;
   private String metalakeEnv;
   private boolean matalakeSet = false;
+  private boolean isMissingNameInfoVisible = true;
+  private boolean isMalformedInfoVisible = true;
```

Comment:
- This smells like an over engineering case. I don't see a solid reason why we want to suppress the error messages.
- Even if we really want to show users that the name is missing or invalid, the flags here are not about whether the messages are "visible" or not. Calling them "visible" is confusing.
- In addition to showing error messages, we'd better let the CLI user know how to fix it. For example, we may want to print "a valid name can contain a-z, A-Z and '.' and '-'." rather than letting them guess what has been wrong. An error message can be more useful than it is now.
-

Reply:
My bad, the scenario is that the error message should be displayed only once, controlled by a variable. Variable names are not clear enough.

Reply:
Well ... I was thinking if we can traverse the control flow to ensure that parameter validation happens once and only once. Or else, we will have a lot of places to do this kind of hacking.

Reply:
Note I'm on holiday for the next two weeks and may be slow in responding. A) this check only needs to be done in one place and B) we have added an error message that says what is missing. Ideally, we would have a better validation system. However, all suggestions along those lines so far have been seen as too complex.